agent/consul/acl.go

```diff
@@ -1078,7 +1082,10 @@ type aclFilter struct {
 // newACLFilter constructs a new aclFilter.
 func newACLFilter(authorizer acl.Authorizer, logger *log.Logger, enforceVersion8 bool) *aclFilter {
 	if logger == nil {
-		logger = log.New(os.Stderr, "", log.LstdFlags)
+		consulLogger := hclog.New(&hclog.LoggerOptions{})
```

Comment:
This should probably explicitly set the Output at least during this massive transition so everything is crystal clear, even if Stderr is the default.